src/components/frame/Extensions/pages/ExtensionDetailsPage/utils.tsx

```diff
@@ -21,6 +21,11 @@ import { type SetStateFn } from '~types/index.ts';
 import { notNull } from '~utils/arrays/index.ts';
 import { addressHasRoles } from '~utils/checks/index.ts';
 
+export enum EXTENSION_TAB_ID {
```

Comment:
Let's use Pascal case here:

```js
export enum ExtensionTabId {}
```

Looking at our codebase, I think we can use this case for constant variables 🤔